browse/services/database/__init__.py

```diff
@@ -298,7 +314,7 @@ def get_hourly_stats(stats_date: Optional[date] = None) -> List:
 
 @db_handle_error(logger=logger, default_return_val=[])
 def get_monthly_submission_stats() -> List:
-    """Get the monthly submission stats."""
+    """Get the monthly submission stats from StatsMonthlySubmission."""
```

Comment:
Not a blocker, but you can make the docs even more magical by using 
```
:class:`.StatsMonthlySubmission`
```
Ref: https://www.sphinx-doc.org/en/1.5/domains.html#python-roles

Reply:
I believe in magic! Thanks--updated.